src/kbc_modules/offline_fs_kbc/README.md

```diff
@@ -11,13 +11,17 @@ The guest must provide OpenSSL at runtime.
 Keys must be provided in the guest file system at `/etc/aa-offline_fs_kbc-keys.json` like:
 ```json
 {
-    "key_id1": "cGFzc3BocmFzZXdoaWNobmVlZHN0b2JlMzJieXRlcyE=",
+    "key_id1": "base64-encoded-key",
+    "key_id2": "base64-encoded-key",
     ...
 }
 ```
 
 with the 32-byte keys base64-encoded.
 
+The script [generate_keys.sh](https://github.com/confidential-containers/attestation-agent/blob/main/sample_keyprovider/src/enc_mods/offline_fs_kbs/generate_keys.sh) can be used to generate some sample keys based on `/dev/random` if it fits your use case sufficiently well.
```

Comment:
Nit: You might want to use backticks in the link names for the two filenames you are linking too for consistency with the `/dev/random` file. That would also be consistent with Kata's doc standards (https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#files-and-command-names).

For example:

```suggestion
The script [`generate_keys.sh`](https://github.com/confidential-containers/attestation-agent/blob/main/sample_keyprovider/src/enc_mods/offline_fs_kbs/generate_keys.sh) can be used to generate some sample keys based on `/dev/random` if it fits your use case sufficiently well.
```

Reply:
thanks @jodh-intel I updated the filenames with between two backticks